packages/codec/lib/types.ts

```diff
@@ -222,18 +219,29 @@ export interface AbiArgument {
    */
   indexed?: boolean; //included for event parameters
   /**
-   * The decoded value of the argument.  Note that this is a [[Format.Values.Result|Values.Result]], so it
+   * The decoded value of the argument.  Note that this is a [[Format.Values.Result|Format.Values.Result]], so it
```

Comment:
You know, if you're making the displayed text the same as the actual link, there's no need to use the bar syntax...